frontend/src/components/common/BackButton.tsx

```diff
@@ -4,16 +4,21 @@ import { Button } from "@chakra-ui/react";
 
 import { ArrowBackOutlineIcon } from "../../assets/icons";
 
-const BackButton = (): React.ReactElement => {
+interface BackButtonProps {
+  size: string;
+  text: string;
+}
+
+const BackButton = ({ size, text }: BackButtonProps): React.ReactElement => {
```

Comment:
maybe we can set default values for props (and make them optional) to avoid changing existing components